platformio.ini

```diff
@@ -53,6 +55,7 @@ build_src_filter =
 
 build_unflags = -std=gnu++11 
 build_flags = 
+  -I include
```

Comment:
You shouldn't need to do this to get it to compile. I forget the details, but I remember having a length conversation with Ben last year about why we shouldn't have this. Let me dig through Teams messages and find it for you.